css/css-regions/META.yml

```diff
@@ -0,0 +1,9 @@
+links:
```

Comment:
This can be removed, css/css-regions/ was removed from WPT recently - https://github.com/web-platform-tests/wpt/pull/21368

Reply:
yes, i will remove it